modules/standard/UtilReplicatedVar.chpl

```diff
@@ -59,38 +63,60 @@ How to use replicated variables:
     // access directly a remote copy on the locale 'remoteLoc' (read or write)
     ... rcRemote(myRepVar, remoteLoc) ...
 
-Advanced: replicate over a subset of locales 'myLocales'.
-// 'myLocales' must be "consistent", as defined for ReplicatedDist.
-Use it as above, except modify the variable declarations as follows:
+For more examples, see :proc:`rcExample` and :proc:`rcExampleOverLocales`
+in the module source code.
```

Comment:
We should avoid telling a documentation reader to visit the module source code, the purpose of documentation is so that we can give them the interface without bogging them down in the details.  In fact, I'm a bit surprised to find examples in module code in the first place, as opposed to the release/examples/primers directory.  If these examples provide benefit in addition to the two you have already provided, consider moving them into the module comment so they are present here.  If that doesn't feel appropriate for doc freeze, then please copy the source code into the comment and "no doc" the examples - that way no code change will occur, but the module documentation appears in a form we like.